### ISSUE TITLE
Add Reactive sample

### ISSUE THREAD

**Issue by AndyDentFree** [realm/realm-dotnet#366]
Swift product [says](https://realm.io/docs/swift/latest/#key-value-observation) it has _a short example of using Realm with ReactiveCocoa from Objective-C, and ReactKit from Swift._.

We need to decide **which** Reactive toolkit to use:

* [ReactiveX](http://reactivex.io/), 
* [ReactiveUI](http://reactiveui.net/) or
* [Redux.NET](https://github.com/GuillaumeSalles/redux.NET)

**Comment by umurgdk** [comment]
I agree. ReactiveUI has more audience then others. Would like to see some examples with it.

**Comment by fealebenpae** [comment]
That's one of the first things I'd like to do as soon as I land https://github.com/realm/realm-dotnet/pull/502/commits/f4169276e3c197a875f2aca259c24e80700344ab. Of course, it'd be great if someone were to make a ReactiveUI facade for this.

**Comment by promontis** [comment]
+1 for ReactiveUI

**Comment by HobDev** [comment]
@AndyDentFree @marukami @ghuntley @umurgdk @fealebenpae @promontis 
I developed a sample app using Xamarin.Forms, ReactiveUI and Realm. The app has a SearchBar and a ListView. User can add items to the listview. User can search the items in the list from `SearchBar` at the top. I got lots of help from ReactiveUI slack channel and stackoverflow to get this to work properly. 
But still this app have some issues. Firstly the items added does not get displayed immediately. When I restart the app then only the added items displayed. Another issue is accessing realm in different methods.
 **I request all the concerned people to contribute and make this sample app perfect example of using Realm with ReactiveUI.**
